### PR TITLE
Task one

### PR DESCRIPTION
**Pull request by ArturKozik**
Check my shittie code, sir

### CODE REVIEW

File: Weather App/Weather App/MainWeatherViewController.swift [hunk 1]
```diff
@@ -8,22 +8,56 @@
...
 class MainWeatherViewController: UIViewController, Storyboarded {
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        firstPointFirst()
+        print("Usage of print in viewDidLoad")
```
Comment (alexsyn02) [inline Weather App/Weather App/MainWeatherViewController.swift:16]: remove this line

File: Weather App/Weather App/MainWeatherViewController.swift [hunk 2]
```diff
@@ -8,22 +8,56 @@
...
+        var int = 54
+        var uint: UInt = 10
+        var float: Float = 50
+        var double = 50.0
+        var string = "Hello Zhopa"
+        print(string)
+        
+        print(int, uint, float, double, string)
+        
+        print("Arthur is learning iOS Development.")
```
Comment (alexsyn02) [inline Weather App/Weather App/MainWeatherViewController.swift:29]: remove this line

File: Weather App/Weather App/MainWeatherViewController.swift [hunk 3]
```diff
@@ -8,22 +8,56 @@
...
+        let minUint = UInt.max
+        let maxUint = UInt.min
+        let minFloat = -Float.greatestFiniteMagnitude
+        let maxFloat = Float.greatestFiniteMagnitude
+        let minDouble = -Double.greatestFiniteMagnitude
+        let maxDouble = Double.greatestFiniteMagnitude
+        
+        print(maxInt, minInt, minUint, maxUint, minFloat, maxFloat, minDouble, maxDouble)
+        
+        print("Arthur is an iOS Dev.")
```
Comment (alexsyn02) [inline Weather App/Weather App/MainWeatherViewController.swift:61]: remove this line

**Review by alexsyn02 — CHANGES_REQUESTED** [review]

File: Weather App/Weather App/MainWeatherViewController.swift (hunk 1 above)
Comment (ArturKozik) [reply Weather App/Weather App/MainWeatherViewController.swift:16]: yes, bo$$

**Pull request merged**